meetup-groups.json

```diff
@@ -45,5 +45,9 @@
   "19781403": {
     "name": "Oxford Hackspace",
     "aliases": ["hack"]
+  },
+  "19683994": {
+    "name": "marlow[dev]",
+    "aliases": ["marlow", "marlow dev"]
```

Comment:
@omgmog: Not really a problem, but the "marlow dev" alias isn't necessary since "marlow dev" contains the substring "marlow", which is the first alias.
I'll try and make the readme and source code clearer.